test/integration/application-proxy/webapp/test/test.qunit.js

```diff
@@ -5,11 +5,24 @@ QUnit.config.autostart = false;
 sap.ui.getCore().attachInit(function() {
 	"use strict";
 
-	sap.ui.require(["test/app/foo"], function() {
-		QUnit.test("Karma", function(assert) {
-			assert.ok(parent.__karma__.files["/base/webapp/.dotfile"], "Karma files should contain dotfiles");
-		});
-
-		QUnit.start();
+	QUnit.test("Karma", function(assert) {
+		assert.ok(parent.__karma__.files["/base/webapp/.dotfile"], "Karma files should contain dotfiles");
```

Comment:
How is this related to the change?

Reply:
It's not. The test has been there before.